src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements FigureDrawer, AreaCalculator {
```

Comment:
```suggestion
public class Circle extends Figure {
```
these interfaces should be accessible through parent class